test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -145,11 +145,11 @@ ucp_ep_h test_ucp_peer_failure::failing_sender() {
 }
 
 ucp_test::entity& test_ucp_peer_failure::stable_receiver() {
-    return m_entities.at(m_entities.size() - 2);
+    return m_entities.at(m_entities.size() - 1);
 }
 
 ucp_test::entity& test_ucp_peer_failure::failing_receiver() {
-    return m_entities.at(m_entities.size() - 1);
+    return m_entities.at(m_entities.size() - 2);
```

Comment:
why needed to change? will it affect existing tests?

Reply:
why not? there are other functions (except smoke_test) which are using failed_receiver, seems failing_receiver and stable_receiver are swapped()?

Reply:
there is no direct access to entries by index (except accessors failed/stable_receiver)... my idea was set stable pair as first sender + last receiver, failed pair set as second sender + (last - 1) receiver. in this case ucp_test::sender & ucp_test::receiver point to stable pair and smoke_test(true) may be reused without initialization of failed pair

Reply:
makes sense to me